Makefile

```diff
@@ -88,6 +88,12 @@ package-deb: build
 docker-build: 
 	docker build -t $(DOCKER_NAMESPACE)/$(COMPONENT):$(VERSION) -f ./cmd/$(COMPONENT)/Dockerfile .
 
+.PHONY: docker-nightly
+docker-nightly:
+	docker buildx create --use --name mybuildx
+	docker buildx build -t public.ecr.aws/aws-otel-test/nightly-build:latest -f ./cmd/$(COMPONENT)/Dockerfile --platform=linux/amd64 --push .
```

Comment:
Isn't the default docker build for amd64? Why is `buildx` needed? Can we just tag the existing docker build? Like https://github.com/aws-observability/aws-otel-collector/blob/27d3ebf590d08a875fb889c011c7b21886d04fbe/.github/workflows/CD.yml#L344-L352

Reply:
Honestly I did this because in the future we may support arm64 since we keep getting requests to add support for it

Reply:
This will make it much easier to add that functionality. May be premature optimization.

Reply:
I feel like that would be a bigger change to docker-build itself. What is the difference between the nightly build and the normal docker build?

Reply:
Just where it publishes the image

Reply:
So, I think we could do what the CD does and just retag the image and push it to the ECR repo. The buildx changes should be made later to the main docker build itself in a separate PR.

Reply:
Agree with @jefchien.

Reply:
Agree with @jefchien